CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Couldnt we just say that we use the same code of conduct as Symfony? Code of conduct is a highly debatable topic. Inventing something new is not really something we should do. That is my 50cents.

Reply:
I'm not for "following" blindly the behavior of "friends" because if they choose to change in a way that means that you're not your own Code of Conduct maintainer. I think that being independent and showing our own choices is better than "following a community". SF CoC is great but I don't like the "Let's follow the other" thing.

Reply:
I understand your argument. 

I also feel it is hard to maintain yet another code of conduct. It is hard to reply, argue and answer questions etc. Im not sure if you followed the 4(?) PRs adding code of conduct in symfony. But there were over 100 of comments for a massive amount of people. They all worked to make it close to perfect. A good code of conduct requires that amount of work. Im not sure Im willing to put down that work here too (no offense). That is why I suggest to use Symfony as a role model.

Reply:
It is not "yet another code of conduct", it's this one:
https://www.contributor-covenant.org/

It's a standard CoC like the PSRs are for php.